analyzer-comments/javascript/elyses-analytic-enchantments/must_avoid_imperative_loop.md

```diff
@@ -0,0 +1,7 @@
+# must avoid imperative loop
+
+Avoid use of for loops with this practice exercise.
+
+According to the introduction, there are built-in methods on the JavaScript Array object that are geared towards analysis.
+
+Take a look at the instructions and/or the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) on what methods are available to use.
```

Comment:
* This line still mentioned instructions instead of introduction.
* The link should go directly to the method list: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#instance_methods